Encode.pm

```diff
@@ -927,6 +929,20 @@ important distinction between C<"UTF-8"> and C<"utf8">.
   encode("utf8",  "\x{FFFF_FFFF}", 1); # okay
   encode("UTF-8", "\x{FFFF_FFFF}", 1); # croaks
 
+This distinction is also important for decoding. In the following,
+C<$s> stores character U+200000, which exceeds UTF-8’s allowed range.
+C<$s> thus stores an invalid Unicode code point:
+
+  $s = decode("utf8", "\xf8\x88\x80\x80\x80");
+
+C<"UTF-8">, by contrast, will either coerce the input to something valid:
+
+    $s = decode("UTF-8", "\xf8\x88\x80\x80\x80"); # U+FFFD
+
+.. or croak:
+
+    decode("UTF-8", "\xf8\x88\x80\x80\x80", 9);
```

Comment:
Hello! Can you put human readable constant names as a third _check_ argument instead of magic number 9?

Reply:
I was shooting for consistency with the other examples above, which give 1 instead of FB_CROAK. Should I update those as well?

Reply:
I've overlooked this.  `9` must be ` DIE_ON_ERR|LEAVE_SRC`.  I'll fix it.

Reply:
Maybe `FB_CROAK` instead of `DIE_ON_ERR` for consistency?

Reply:
fixed in https://github.com/dankogai/p5-encode/commit/67f59dd4092c12043182e427dfaa5abd5c791e58